tests/waku_filter_v2/test_waku_filter.nim

```diff
@@ -11,360 +11,315 @@ import
-
-    let
-      response1 = await wakuFilterClient.subscribe(serverSwitch.peerInfo.toRemotePeerInfo(),
-                                                   pubsubTopic,
-                                                   contentTopics)
-
-      response2 = await wakuFilterClient.subscribe(serverSwitch.peerInfo.toRemotePeerInfo(),
-                                                   pubsubTopic2,
-                                                   contentTopics)
-
-    # Then
-    check:
-      response1.isOk()
-      response2.isOk()
-      wakuFilter.subscriptions.len == 1
-      wakuFilter.subscriptions.hasKey(clientPeerId)
-
-    ## Step 2: We receive messages for multiple subscribed pubsub topics and content topics
-
-    # When
-    let msg1 = fakeWakuMessage(contentTopic=DefaultContentTopic)
-    await wakuFilter.handleMessage(DefaultPubsubTopic, msg1)
-
-    require await pushHandlerFuture.withTimeout(3.seconds)
-
-    # Then
-    let (pushedMsgPubsubTopic, pushedMsg) = pushHandlerFuture.read()
-    check:
-      pushedMsgPubsubTopic == DefaultPubsubTopic
-      pushedMsg == msg1
-
-    # When
-    let msg2 = fakeWakuMessage(contentTopic=contentTopic2)
-    pushHandlerFuture = newFuture[(string, WakuMessage)]() # Clear previous future
-    await wakuFilter.handleMessage(pubsubTopic2, msg2)
-
-    require await pushHandlerFuture.withTimeout(3.seconds)
-
-    # Then
-    let (pushedMsgPubsubTopic2, pushedMsg2) = pushHandlerFuture.read()
-    check:
-      pushedMsgPubsubTopic2 == pubsubTopic2
-      pushedMsg2 == msg2
-
-    ## Step 3: We can selectively unsubscribe from pubsub topics and content topic(s)
-
-    # When
-    let response3 = await wakuFilterClient.unsubscribe(serverSwitch.peerInfo.toRemotePeerInfo(),
-                                                       pubsubTopic2,
-                                                       @[contentTopic2])
-    require response3.isOk()
-
-    let msg3 = fakeWakuMessage(contentTopic=contentTopic2)
-    pushHandlerFuture = newFuture[(string, WakuMessage)]() # Clear previous future
-    await wakuFilter.handleMessage(pubsubTopic2, msg3)
-
-    # Then
-    check:
-      not (await pushHandlerFuture.withTimeout(2.seconds)) # No message should be pushed
-
-    ## Step 4: We can still receive messages for other subscribed pubsub topics and content topics
-
-    # When
-    pushHandlerFuture = newFuture[(string, WakuMessage)]() # Clear previous future
-    await wakuFilter.handleMessage(DefaultPubsubTopic, msg3)
-
-    require await pushHandlerFuture.withTimeout(3.seconds)
-
-    # Then
-    let (pushedMsgPubsubTopic3, pushedMsg3) = pushHandlerFuture.read()
-    check:
-      pushedMsgPubsubTopic3 == DefaultPubsubTopic
-      pushedMsg3 == msg3
-
-    # Teardown
-    await allFutures(wakuFilter.stop(), wakuFilterClient.stop(),
-                     serverSwitch.stop(), clientSwitch.stop())
+  discard
+  # asyncTest "simple subscribe and unsubscribe request":
```

Comment:
I see prev comment from Ivan and the answer but am I missed something or why not simple remove it fully... let git remember if something is needed later/parallel?

Reply:
This was a mistake I made, it's removed in one of the following PRs.